combinations/hash.tsv

```diff
@@ -172,3 +172,5 @@ gatk=3.5,samtools=1.13
 requests=2.26	quay.io/bioconda/base-glibc-debian-bash:latest	0
 mirtop=0.4.24,samtools=1.13,r-base=4.1.1,r-data.table=1.14.2
 krakenuniq=0.5.8,pigz=2.6
+gatk4=4.2.0.0,R=4.1,r-ggplot2=3.3.5,datamash=1.1.0
+jemunro::quanttb=1.01,bioconda::bwa=0.7.17
```

Comment:
However, for the `quanttb` package - would the build system pick it up without the `jemunro` ?

Reply:
Oh I see, no. But we also do not support that. Please move the package to bioconda or conda-forge.

Reply:
Got it, so for the time being, I'll remove that line from the PR.